packages/extensions/src/table/style.css

```diff
@@ -33,3 +33,7 @@
   background-color: hsla(var(--color), 20%);
   border: 1px double hsl(var(--color));
 }
+.ProseMirror th > p,
```

Comment:
It's possible to have another type of nodes than `<p>` (e.g. `<h1>`) inside a cell. Maybe we should add the padding to `<td>` instead of to `<p>`.